.gitignore

```diff
@@ -10,3 +10,4 @@ tmp
 Thumbs.db
 /platform
 /i18n
+plugins/ti.alloy/hooks/alloy.js
```

Comment:
We should do this for the Alloy app template as well, so all new apps don't suffer this. It may also be a good chance to revisit the general .gitignore of Alloy- and classic-apps.